grammar.js

```diff
@@ -182,7 +182,15 @@ module.exports = grammar({
     constant_type_argument: ($) => $._constant_type,
 
     external_type: ($) =>
-      seq(optional($.visibility_modifier), "external", "type", $.type_name),
+      prec.right(
+        seq(
+          optional($.visibility_modifier),
+          // DEPRECATED: the external token was removed in v0.30.
+          optional("external"),
```

Comment:
I'm thinking here that we may want to accept the old syntax for some amount of time after some syntax is changed. (A few minor releases? I'm not sure how long.) What do you think?

Reply:
Sounds fab to me.